### PR TITLE
Add lmp minimum date validation to USSD registration

### PR DESCRIPTION
**Pull request by rudigiesler**
This PR adds the configurable minimum date validation, as well as removes the months from the selection list that do not fall within the minimum date.

### CODE REVIEW

File: go-ussd_registration.js [hunk 1]
```diff
@@ -1708,14 +1719,16 @@ go.app = function() {
         // PaginatedChoiceState st-05
         self.add('state_last_period_month', function(name) {
             var today = go.utils.get_today(self.im.config);
+            var minimum_weeks = (self.im.config.minimum_weeks || 11);
```
Comment (erikh360) [inline go-ussd_registration.js:1722]: Where is this default value set?

**Review by erikh360 — COMMENTED** [review]
Looks good, just the one comment and a question: Shouldn't we be doing this for voice too?

File: go-ussd_registration.js (hunk 1 above)
Comment (rudigiesler) [reply go-ussd_registration.js:1722]: This looks in the config for the `minimum_weeks` key, and if it's not there, defaults to 11. I'm not sure if there's a better way of doing a default value in the jsbox apps.

Comment (erikh360) [reply go-ussd_registration.js:1722]: Ok, cool!

**Comment by rudigiesler** [comment]
@erikh360 Yes, I will do the voice registration app in a separate PR.

**Review by erikh360 — APPROVED** [review]

**Pull request merged**